### PR TITLE
docs: Timeouts in CTP

### PR DESCRIPTION
**Pull request by zirain**
Fixes: https://github.com/envoyproxy/gateway/issues/2676

### CODE REVIEW

File: tools/src/helm-docs/go.mod [hunk 1]
```diff
@@ -1,10 +1,8 @@
 module github.com/envoyproxy/gateway/tools/src/helm-docs
 
-go 1.22
+go 1.21
 
-toolchain go1.22.0
-
-require github.com/norwoodj/helm-docs v1.13.0
```
Comment (zirain) [inline tools/src/helm-docs/go.mod:7]: `v1.13` require `go1.22`, revert first.

File: site/content/en/latest/user/client-traffic-policy.md [hunk 2]
```diff
@@ -410,5 +410,58 @@ Handling connection for 8888
 }
 ```
 
+### Enable HTTP request timeout
```
Comment (arkodg) [inline site/content/en/latest/user/client-traffic-policy.md:413]:
```suggestion
### Enable HTTP Request Received timeout
```

File: site/content/en/latest/user/client-traffic-policy.md [hunk 3]
```diff
@@ -410,5 +410,58 @@ Handling connection for 8888
...
+
```
Comment (arkodg) [inline site/content/en/latest/user/client-traffic-policy.md:414]:
```suggestion
This feature allows you to limit the take taken by the Envoy Proxy fleet to receive the entire request from the client, which is useful in preventing certain clients from consuming too much memory in Envoy
```

**Review by arkodg — APPROVED** [review]

**Review by shawnh2 — APPROVED** [review]

**Pull request merged**